src/main.dev.js

```diff
@@ -248,6 +248,14 @@ app.on('ready', async () => {
     }
   });
 
+  // Ensure the application window has focus as well as the emmbedded content
+  // will be called on settings page and when url is switched to home url
+  mainWindow.webContents.on('dom-ready', () => {
+    console.log('DOM READY');
+    mainWindow.focus();
```

Comment:
We should add a wrapper around this to only fire on production. See some previous commits for history.
https://github.com/scimusmn/stele/commit/8b81666d74fb84a4fafa02a90a1c891d05172cf1

Essentially we don't want to re-introduce this bug: https://github.com/scimusmn/stele/issues/23